### PR TITLE
add desciption of bfloat type

### PR DESCRIPTION
**Pull request by stuij**

### CODE REVIEW

File: abi/aapcs64/aapcs64.rst [hunk 1]
```diff
@@ -466,11 +466,15 @@ Table 1, Byte size and byte alignment of fundamental data types shows the fundam
 Half-precision Floating Point
 -----------------------------
 
-The architecture provides hardware support for half-precision values. Two formats are currently supported: the format specified in IEEE 754-2008 and an Alternative format that provides additional range but has no NaNs or Infinities. This base standard of the AAPCS64 specifies two variants:
+The architecture provides hardware support for half-precision values. Three formats are currently supported:
 
-- The SysV-like variants use the IEEE 754-2008 defined format.
+1 - half-precision format specified in IEEE754-2008
```
Comment (momchil-velikov) [inline abi/aapcs64/aapcs64.rst]: This numbering here, shouldn't it be done with some markup thing?

**Comment by momchil-velikov** [comment]
The patch does not contain name mangling for `__bf16`, despite the commit summary.

File: abi/aapcs64/aapcs64.rst (hunk 1 above)
Comment (stuij) [reply abi/aapcs64/aapcs64.rst]: Ah thanks, used the wrong syntax. Changed.

**Comment by stuij** [comment]
> The patch does not contain name mangling for `__bf16`, despite the commit summary.

thanks, removed name mangling text.

**Pull request merged**